pkg/kubelet/eviction/helpers_test.go

```diff
@@ -1289,7 +1289,7 @@ func TestMakeSignalObservations(t *testing.T) {
 				},
 				// Used for memory signal observations on windows
 				{
-					Name: statsapi.SystemContainerPhysicalMemory,
+					Name: statsapi.SystemContainerWindowsGlobalCommitMemory,
```

Comment:
nit: Can we make the commits additive?

Reply:
I'll rebase into a fewer commits.
I had let some of the iterative commits in to facilitate discussion in sig-windows but now that we have reached agreement this can and should go in as a single commit.